src/actions/mod.rs

```diff
@@ -202,7 +202,7 @@ impl ActionHandler {
 
         let rustw_handle = thread::spawn(move || {
             let file_name = parse_file_path(&doc.text_document.uri).unwrap();
-            let symbols = analysis.symbols(&file_name).unwrap_or(vec![]);
+            let symbols = analysis.symbols(&file_name).unwrap_or_else(|_| vec![]);
```

Comment:
The  `|_|` takes one `()` (unit) argument. `String::new` doesn't. This is unfortunately not the same.